File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
 
 {% endtabs %}
 
+## Recreate Nested Metafile
+
+This setting allows you to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
+This property is recommended to resolve the scaling problem of nested metafile images by regenerating the nested metafile images present in the PowerPoint presentation document.
+
+The following code sample shows how to use this property to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Open a PowerPoint Presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+    //Create an instance of the ChartToImageConverter and assign it to the ChartToImageConverter property of the Presentation.
+    pptxDoc.ChartToImageConverter = new ChartToImageConverter();
+    //Initialize the conversion settings.
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    //Set RecreateNestedMetafile property to true to Recreate the Nested Metafile automatically.              
```

Comment:
//Set the RecreateNestedMetafile property to true to recreate the Nested Metafile automatically.

Reply:
Feedback addressed